packages/wrangler/src/versions/deploy.ts

```diff
@@ -205,6 +209,68 @@ function getConfig(
 	return config;
 }
 
+/**
+ * Prompts the user for confirmation when overwriting the latest deployment, given that it's split.
+ */
+export async function confirmLatestDeploymentOverwrite(
+	accountId: string,
+	scriptName: string
+) {
+	try {
+		const latest = await fetchLatestDeployment(accountId, scriptName);
+		if (latest && latest.versions.length >= 2) {
+			// Fetch the version details.
+
+			const versionCache: VersionCache = new Map();
+			const versionIds = latest.versions.map((v) => v.version_id);
+			await fetchVersions(accountId, scriptName, versionCache, ...versionIds);
+
+			// Format each version.
+
+			const formattedVersions = latest.versions.map((traffic) => {
+				const version = versionCache.get(traffic.version_id);
+				assert(version);
+				const percentage = brandColor(`(${traffic.percentage}%)`);
+				const details = formatLabelledValues(
+					{ Created: new Date(version.metadata["created_on"]).toISOString() },
+					{
+						indentationCount: 4,
+						labelJustification: "right",
+						formatLabel: (label) => gray(label + ":"),
+						formatValue: (value) => gray(value),
+					}
+				);
+				return `${percentage} ${version.id}\n${details}`;
+			});
+
+			// Format deployment.
+
+			const formattedDeployment = formatLabelledValues({
+				"Version(s)": formattedVersions.join("\n\n"),
+			});
+
+			// Print message and confirmation.
```

Comment:
Can this all be replaced with a call to printLatestDeployment (the function right below this one)?